src/pages/create/Activity.tsx

```diff
@@ -138,11 +170,15 @@ export const CreateActivity = () => {
       client,
       activityTypes.find((e) => e.label === type)?.value as string,
       data
-    );
+    )
+      .then(() => navigate(-1))
+      .catch((e: Error) => {
+        setSubmissionError(mapErrorMessage(e));
 
-    navigate(-1);
+        setIsSubmitting(false);
+      });
   };
-
+  console.log(errors);
```

Comment:
think this is debug? please remove

Reply:
@iTweeno could you also see if we can lint for use of `console.*()`? Could do with CI catching this sort of thing...